src/Tuura/Concept/FSM/Translation.hs

```diff
@@ -116,21 +121,28 @@ translate circuit signs =
 
       Invalid errs -> addErrors errs
 
+-- Find the numeric value of the initial state.
 getInitialState :: CircuitConcept a -> [a] -> Int
 getInitialState circuit signs = encToInt state
   where
     state = map (fromBool  . getDefined . initial circuit) signs
 
+-- From a boolean, return the Tristate version.
+-- Used to find Tristate encodings of before and after states of signals.
 fromBool :: Bool -> Tristate
 fromBool x = if x then triTrue else triFalse
 
+-- Create consistency loops, ensuring a signal can transition both high and low
```

Comment:
Let's avoid using `consistency loop` as a term. Something like `signal transition loop` sounds clearer for people who are not intimately familiar with STG properties.